### DISCUSSION TITLE
Eclipse wizard does not appear

### DISCUSSION THREAD

**Discussion by simsla987** [ThalesGroup/xsmp-modeler-core#32] [Q&A] [answered]
Hello,
I'm following the wiki : https://github.com/ThalesGroup/xsmp-modeler-core/wiki/User-guide#installation

I have a fresh install of Eclipse.
Version: 2023-06 (4.28.0)
Build id: 20230608-1333

I added the jar in the Available software Sites :
full url : jar:https://github.com/ThalesGroup/xsmp-modeler-core/releases/download/v1.1.0-rc2/org.eclipse.xsmp.repository-v1.1.0-rc2.zip!/
![image](https://github.com/ThalesGroup/xsmp-modeler-core/assets/71452181/27b3358c-2694-47f7-a11c-c58ef4353b48)

Now, I want to create an Xsmp project but i don't see the wizard int Project... : 
Run the wizard from File → New → Project... → Xsmp → Xsmpcat Project and follow the instructions.
![image](https://github.com/ThalesGroup/xsmp-modeler-core/assets/71452181/d6d6c049-e129-416c-8ca6-16e7cf0bf9e4)

How can I create the project ?
Of course I tried to restart Eclipse

**Comment by ydaveluy** [comment, marked as answer]
Hello, it seems that the XSMP plugin is not installed.

You have to install it with : Help -> Install new software...

In "work with" select the xsmp update site (the jar file), then select the Xsmp plugins to be installed.

After a restart of Eclipse you should have access to the wizard.

**Reply by simsla987** [reply]
Thanks !
I confirm that i have the wizard with that solution :)

**Comment by ydaveluy** [comment]
I updated the procedure in the wiki
